postgres-appliance/configure_spilo.py

```diff
@@ -219,14 +217,21 @@ def get_instance_metadata(provider):
                 'id': socket.gethostname(),
                 'zone': 'local'}
 
+    if USE_K8S:
```

Comment:
Yes, please use "Kubernetes" everywhere for consistency and reader convenience, i.e. `USE_KUBERNETES` here.